ImplementationGuide/markdown/UebergreifendeFestlegungen/UebergreifendeFestlegungen_Suchparameter.md

```diff
@@ -24,15 +30,21 @@ Für die im Rahmen dieses Leitfadens relevanten Typen gelten folgende allgemeine
 
 Die Präfixe `lt`,`le`,`gt`,`ge`,`eq` MÜSSEN für jeden Suchparameter vom Typ 'date/dateTime' unterstützt werden.
 
+Begründung: Die Funktionalität datums-eingeschränkt suchen zu können ist essentiell.
+
+Hinweis: Die Abfragemöglichkeit arbeitet ungewollten Massendatenabfragen entgegen, da sich sonst Suchen zwangsläufig über den gesamten Zeitraum erstrecken würden.
+
+
+
 **Beispiele**:
 
-```[base]/Patient?birthDate=ge2000-01-01``` <br>
-Suche nach allen Patienten mit einem Geburtsdatum 2000-01-01T00:00 oder später.
+```[base]/Encounter?date=eq2024-01-01&patient=Patient/Test``` <br>
```

Comment:
question: @simoneOnFhir haben die "<br> eine Funktion oder sind das c+p-errors?

Reply:
[Das ist Markdown für "hier kommt ein Codeblock"](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks)

Reply:
@simoneOnFhir ich meinte die \<br>

Reply:
... gehe aber davon, auss dass wir einfach ein breakline zwischen den examples wollen. Macht sinn.